src/CoreBundle/Controller/ChameleonControllerResolver.php

```diff
@@ -49,14 +49,13 @@ public function __construct(ContainerInterface $container, ControllerResolver $d
 
     /**
      * {@inheritdoc}
-     *
-     * @return callable|false|null
      */
     public function getController(Request $request)
     {
+        /** @var object|null|string $controller */
         $controller = $request->attributes->get('_controller', null);
         if (null === $controller) {
-            return null;
+            return false;
```

Comment:
Symfony expects us to return `callable|false` where this method used to return `callable|false|null`. I changed the null-return to false here. I don't think that this should have behavioural impact